app/models/book.rb

```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+	validates_presence_of :title, :category
```

Comment:
We also need validation on an attribute that lets us know the progress of the book (i.e, how much percentage read is the book already).

Reply:
For some reason, if I add :read to validates_presence_of, everything just stops working. Both saving and updating books returns an error. Can you help with this?